titanium-rdf-api/src/test/scala/eu/ostrzyciel/jelly/convert/titanium/TitaniumJellyEncoderSpec.scala

```diff
@@ -0,0 +1,41 @@
+package eu.ostrzyciel.jelly.convert.titanium
+
+import eu.ostrzyciel.jelly.core.{JellyOptions, Constants as CoreConstants}
+import eu.ostrzyciel.jelly.core.proto.v1.{LogicalStreamType, PhysicalStreamType}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.jdk.CollectionConverters.*
+
+/**
+ * Tests for the auxiliary methods of the TitaniumJellyEncoder.
+ * The main tests are done in the integration-tests module.
+ */
+class TitaniumJellyEncoderSpec extends AnyWordSpec, Matchers:
+  "TitaniumJellyEncoder" should {
+    "be created with default options" in {
+      val encoder = TitaniumJellyEncoder.factory()
+      encoder.getOptions should be (
+        JellyOptions.smallStrict
+          .withPhysicalType(PhysicalStreamType.QUADS)
+          .withLogicalType(LogicalStreamType.FLAT_QUADS)
+          .withVersion(CoreConstants.protoVersion_1_0_x)
+      )
+      encoder.getRowCount should be (0)
```

Comment:
may be good to add some test cases when the rowcount is larger than 0

Reply:
OK, done.